cicd/2-cicd/cicd.template.yml

```diff
@@ -25,7 +25,7 @@ Parameters:
   CodeStarConnectionResourceId:
     Type: String
     Description: The Resource Id component of the CodeStar connection ARN for the code-dot-org GitHub
-    Default: 9e27ebd6-de25-495c-9a2d-b24077376de8
+    Default: 8ec605d9-574e-4f33-9fca-86831566791c
```

Comment:
Did we rotate the codestar/github connection?  Which user do we set those up under BTW.  I wanted to see what access the connection had in GitHub

Reply:
I needed to recreate it to fix a bug. The Codestar connection must be manually created, and the current one is set up under my user, cat5inthecradle (erring on the side of 'easy to fix if I leave' vs spending time setting up the right service account. The user 'deploy-code-org' might be suitable, but I don't think it has enough permissions). It only has access to javabuilder and aiproxy.